### ISSUE TITLE
Add names to some of the files

### ISSUE THREAD

**Issue by v-in-cube** [open-reaction-database/ord-data#132]
During loading the databases instances, I found that certain instances do not have names.

```
    url = "https://github.com/open-reaction-database/ord-data/blob/main/data/"+link+"?raw=true"
    print(url)
    pb = wget.download(url)
    print(pb)
    data = message_helpers.load_message(pb, dataset_pb2.Dataset)
    list_names.append(data.name)
```

Currently, there are 6 databases that have empty names, could it be fixed, please?

```
ord_dataset-0c75d67751634f0594b24b9f498b77c2.pb.gz
ord_dataset-3b5db90e337942ea886b8f5bc5e3aa72.pb.gz
ord_dataset-4d431564f3ef4e9c91d8da5836f4eae6.pb.gz
ord_dataset-68cb8b4b2b384e3d85b5b1efae58b203.pb.gz
ord_dataset-cbcc4048add7468e850b6ec42549c70d.pb.gz
ord_dataset-d26118acda314269becc35db5c22dc59.pb.gz
```

**Comment by skearnes** [comment]
Thanks @v-in-cube; I'll take a look.

**Comment by skearnes** [comment]
Fixed by #191
